### PR TITLE
rbx fix and added bash_functions to prereq

### PR DESCRIPTION
**Pull request by nbrady-techempower**
This pr forces `rbx-2.4` and `rbx-2.5` to use `clang-3.4`. `clang-3.5` is being installed by the prerequisites, though it looks as though only silicon uses it. Long term, all gcc/clang compilers should be moved out of prereqs and into systools. #2088 

This also adds the `bash_functions.sh` source to `prerequisites.sh` for use of `fw_exists`. #2092

### CODE REVIEW

**Comment by knewmanTE** [comment]
LGTM!

File: frameworks/PHP/cygnite-php-framework/apps/views/bench/fortunes.view.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html>
-<head><title>Fortunes</title></head>
+<head>
+    <title>Fortunes</title>
+</head>
+
```
Comment (msmith-techempower) [inline frameworks/PHP/cygnite-php-framework/apps/views/bench/fortunes.view.php]: For future reference, we should all strive to not make changes outside of the PR's goal. I didn't see anything about `cygnite` in the original comment. Additionally, why add bytes when the purpose of the test is to show the performance of the platform? They add up over thousands of requests and are unnecessary.

File: toolset/setup/linux/languages/rbx-2.5.sh [hunk 2]
```diff
@@ -12,6 +12,9 @@ RETCODE=$(fw_exists ${IROOT}/rbx-2.5.installed)
 # update this when it changes
 RBX_VERSION=2.5.2
 
+# This version of RBX needs clang 3.4
+sudo apt-get -yq --force-yes install clang-3.4
```
Comment (msmith-techempower) [inline toolset/setup/linux/languages/rbx-2.5.sh]: Ugh... more apt-get, more sudo.

@nbrady-techempower if you have time, can you quickly experiment to see if you can make `clang-3.4` into its own systool dependency and test out `fw_apt_to_iroot`? I would love to move away from as many of these apt-get as possible.

File: frameworks/PHP/cygnite-php-framework/apps/views/bench/fortunes.view.php (hunk 1 above)
Comment (knewmanTE) [inline frameworks/PHP/cygnite-php-framework/apps/views/bench/fortunes.view.php]: @msmith-techempower Nate and I were talking about this yesterday. We think this was a discrepancy where a few minor commits had been merged into master and were in Nate's branch, but hadn't been pulled into round-14 yet. In the future we'll be sure to update round-14 with the latest from master to ensure that this is the case.

File: toolset/setup/linux/languages/rbx-2.5.sh (hunk 2 above)
Comment (nbrady-techempower) [inline toolset/setup/linux/languages/rbx-2.5.sh]: @msmith-techempower Sure thing!

**Pull request merged**